### PR TITLE
feat:Corrected LEADING_PRODUCT_REPOSITORY_URL

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description
Corrected the LEADING_PRODUCT_REPOSITORY_URL

### CODE REVIEW

File: .tractusx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 product: "sldt-digital-twin-registry"
```
Comment (bs-sili) [inline .tractusx:1]: Could you please correct from "sldt-digital-twin-registry" to "Digital Twin Registry"? Then we have the same wording like in our other services.

Comment (shijinrajbosch) [reply .tractusx]: done

**Review by bs-sili — APPROVED** [review]
LGTM

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**